index.css

```diff
@@ -0,0 +1,93 @@
+*{
+    border-color: rgba(0,0,0,0);
+    border-style: solid;
+    margin: 0;
+    padding: 0p;
+}
+body{
+    display: block;
+    border-width: 0px 33vw;
```

Comment:
Ohhh, I understand now. Sneaky but, damn no, create a wrapper (container) inside body, place the content in it, and wrap it the way I showed you.

Reply:
Actually, it is fine. I forgot to say you are allowed to use `margin-*` for containers. My bad.